src/core/server/core_route_handler_context.ts

```diff
@@ -75,19 +76,23 @@ class CoreSavedObjectsRouteHandlerContext {
     return this.#typeRegistry;
   }
 
-  public get exporter() {
+  public getClient = (options?: SavedObjectsClientProviderOptions) => {
+    return this.savedObjectsStart.getScopedClient(this.request, options);
+  };
+
+  public getExporter = (client: SavedObjectsClientContract) => {
     if (this.#exporter == null) {
-      this.#exporter = this.savedObjectsStart.createExporter(this.client);
+      this.#exporter = this.savedObjectsStart.createExporter(client);
     }
     return this.#exporter;
-  }
+  };
```

Comment:
I don't think we should be memoizing the exporter/importer any more, as the method could be called with distinct clients?

Reply:
the request context API's are meant to be convenience API's that are already scoped to the incoming request, imo if we're not scoping anything to the request we shouldn't add it to the request context.

So now that we're requiring a client to be provided, I see little value in having these and plugins could just use `savedObjectsStart.createExporter`?